research-software/elk/elk.rst

```diff
@@ -62,17 +64,63 @@ MPI ELK jobs
    #SBATCH --qos=standard
 
    # Load the elk module
-   # Launch the executable
+   # Launch the executable 
+   # Input filename elk.in
+
+   module -s restore /etc/cray-pe.d/PrgEnv-gnu
+   module load elk
 
-   module load elk/1.2.3.4
-   srun elk MY_ELK_INPUT.in
+   srun elk 
 
 
 Mixed MPI/OpenMP ELK jobs
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+The following script will run an ELK job on 4 nodes, using 8 OpenMP threads and 16 MPI tasks per node.
+
+.. warning::
+
+  This script is provisional and requires verification
+
+::
+
+   #!/bin/bash
+
+   # Request 512 MPI tasks (4 nodes at 128 tasks per node) with a
```

Comment:
I would edit that line to reflect that you request _4 nodes, using 8 OpenMP threads and 16 MPI tasks per node_.

Reply:
Well spotted! I've made the suggested change